### ISSUE TITLE
Will there be a 1.21 version?

### ISSUE THREAD

**Issue by 52nomis9** [someElseIsHere/SeedCrackerX-Forge#3]
Thank you very much for your contribution and sharing. Do you have any plans to launch the 1.21 version?

**Comment by Jelasdongsip** [comment]
Oui, j'aimerais aussi savoir si une version 1.21 doit arriver dans le futur
